data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/AbstractCinemaScraper.java

```diff
@@ -143,7 +151,8 @@ private Movie retrieveMovieInformation(Movie movie) {
         try {
             return tmdb.getMovie(movie);
         } catch (MovieDbException e) {
-            //Movie DB does not know this movie for some reason. Basically this means we won't show it
+            // Movie DB does not know this movie for some reason. Basically this
+            // means we won't show it
```

Comment:
This smells like unnecessary change, but I think I'm going to allow it because the space at the beginning makes it prettier